packages/go_router_builder/example/lib/all_types.dart

```diff
@@ -0,0 +1,160 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
I think I know what you are trying to show in this example, but it is a bit too made up. maybe we can have a simpler version of this example that show a route with combination of route parameters and query parameters

Reply:
This is meant as a psedo-test. To ensure all supported types keep working. 😄  See the other examples.